### PR TITLE
Internal: Added maps, text_path, social icons template tests [ED-10493]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: Added maps, text_path, social icons template tests

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/elements-regression/tests/elements-regression.test.js [hunk 1]
```diff
@@ -40,7 +40,11 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 			await editorPage.closeNavigatorIfOpen();
 			await editorPage.loadTemplate( filePath );
 			if ( 'video' === widgetType ) {
-				await editorPage.waitForVideoLoaded();
+				await editorPage.waitForIframeToLoaded( { iframeSelector: EditorSelectors.videoIframe, elementToWaitFor: EditorSelectors.playIcon } );
```
Comment (yotamselementor) [inline tests/elements-regression/tests/elements-regression.test.js:43]: Maybe this code should be in waitForIframeToBeLoaded? I'm not sure it should, but we repeat the same code for both the published and unpublished parts

Comment (dmaziar) [reply tests/elements-regression/tests/elements-regression.test.js:43]: It's not the same. Video and google maps have different iframes. Also before publishing we have elementor-preview-iframe after publish it is missing. waitForIframeToLoaded is just reused 4 times. 
Maybe I can find a better solution

Comment (dmaziar) [reply tests/elements-regression/tests/elements-regression.test.js]: fixed

File: tests/elements-regression/tests/elements-regression.test.js [hunk 2]
```diff
@@ -39,25 +55,26 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
...
 			await editorPage.publishAndViewPage();
 			await editorPage.waitForElementRender( widgetIds[ 0 ] );
-			if ( 'video' === widgetType ) {
-				await editorPage.waitForVideoLoaded( true );
-			}
-			await expect( page.locator( EditorSelectors.container ) ).toHaveScreenshot( `${ widgetType }_published.png`, { maxDiffPixels: 100 } );
+			await editorPage.waitForIframeToLoaded( widgetType, true );
+			await response;
+
+			await expect( page.locator( EditorSelectors.container ) ).toHaveScreenshot( `${ widgetType }_published.png`, { maxDiffPixels: 100 }, { timeout: 10000 } );
```
Comment (hein-obox) [inline tests/elements-regression/tests/elements-regression.test.js:77]: @dmaziar Can you please check this?

Screenshot 2023-04-25 at 15 12 13

Comment (yotamselementor) [reply tests/elements-regression/tests/elements-regression.test.js:77]: This shows that the Google maps test was flaky-can you please check it out?

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: tests/playwright/pages/editor-page.js [hunk 3]
```diff
@@ -681,14 +681,23 @@ module.exports = class EditorPage extends BasePage {
...
 		if ( isPublished ) {
-			await this.page.frameLocator( EditorSelectors.videoIframe ).nth( 0 ).locator( EditorSelectors.playIcon ).waitFor();
+			await this.page.frameLocator( frames[ widgetType ][ 0 ] ).last().locator( frames[ widgetType ][ 1 ] ).waitFor();
 		} else {
 			const frame = this.getPreviewFrame();
 			await frame.waitForLoadState();
-			await frame.waitForSelector( EditorSelectors.videoIframe );
-			await frame.frameLocator( EditorSelectors.videoIframe ).nth( 0 ).locator( EditorSelectors.playIcon ).waitFor();
+			await frame.waitForSelector( frames[ widgetType ][ 0 ] );
+			await frame.frameLocator( frames[ widgetType ][ 0 ] ).nth( 0 ).locator( frames[ widgetType ][ 1 ] ).waitFor();
```
[the remark this answers is not in the document]
Comment (dmaziar) [reply tests/playwright/pages/editor-page.js]: fixed

File: tests/elements-regression/tests/elements-regression.test.js (hunk 2 above)
Comment (dmaziar) [reply tests/elements-regression/tests/elements-regression.test.js:77]: fixed

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**